platform/mock-radio.ts

```diff
@@ -34,10 +34,17 @@ export class MockNetwork {
   ) {
     const key = address + ':' + port.toString();
     if (this.udpListeners.has(key)) {
-      this.udpListeners[key].push(listener);
+      const listeners = this.udpListeners.get(key);
+      if (listeners !== undefined) {
+        listeners.push(listener);
+      }
+      const tlisteners = this.udpListeners.get(key);
+      if (tlisteners !== undefined) {
+        console.log(tlisteners.length);
+      }
       return;
     }
-    this.udpListeners[key] = [listener];
+    this.udpListeners.set(key, [listener]);
```

Comment:
Should we overwrite this or push?

Reply:
Should overwrite, but this was missing an early return.  Added the early return.  As a side note I may replace all this in #26